test/e2e/wallet-helpers/vega-api.ts

```diff
@@ -0,0 +1,85 @@
+import { TransferRequest } from '@vegaprotocol/protos/dist/vega/TransferRequest'
+import { WebDriver } from 'selenium-webdriver'
+import { VoteSubmission } from '@vegaprotocol/protos/dist/vega/commands/v1/VoteSubmission'
+import { DelegateSubmission } from '@vegaprotocol/protos/dist/vega/commands/v1/DelegateSubmission'
+interface Key {
+  index: number
+  name: string
+  publicKey: string
+}
+
+export class VegaAPI {
+  private driver: WebDriver
+
+  constructor(driver: WebDriver) {
+    this.driver = driver
+  }
+
+  async connectWallet() {
+    return await this.driver.executeScript<string>(async () => {
+      if (!window.vega) {
+        throw new Error('Vega wallet not found')
```

Comment:
chore (blocking): error message isn't strictly speaking correct, in Eth land there are ways of injecting content scripts without wallets and that is also true in Vega (for example the view as function). Suggest change to `vega not detected` or `content script not found`.